README.md

```diff
@@ -146,12 +146,20 @@ class Service
     """
 ```
 
+#### global_yaml
+Call this _on the top level of your Tiltfile_ on a string of YAML.
+
+We will infer what (if any) of the k8s resources defined in your YAML correspond to `Services` defined elsewhere in your Tiltfile (matching based on the DockerImage ref and on pod selectors). Any remaining YAML is _global YAML_, i.e. YAML that Tilt applies to your k8s cluster independently of any `Service` you define. 
+```python
+def global_yaml(yaml: string) -> None
+```
+
 #### k8s_service
 
-Creates a kubernetes service that tilt can deploy using the yaml text and the image passed in.
+Creates a kubernetes service that tilt can deploy using the the image passed in. Optionally, you may also pass the Kubernetes resource YAML. If the YAML is not passed, we expect to be able to extract it from `global_yaml` (see above).
```

Comment:
Will we error here if we can't find it?

Reply:
good Q. i believe we won't error right here, but later on, when we call manifest.Validate()